src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    public Square(Colors color, double side) {
```

Comment:
You should avoid using magic numbers or expressions directly in the constructor call. It's better to define constants for default values and use them, or ensure the values are passed in from an external caller, such as `FigureSupplier`.